stdlib/public/core/UnsafeRawPointer.swift

```diff
@@ -473,8 +483,6 @@ public struct UnsafeRawPointer: _Pointer {
       )
       return temporary.pointee
     }
-    //FIXME: reimplement with `loadRaw` when supported in SIL (rdar://96956089)
-    // e.g. Builtin.loadRaw((self + offset)._rawValue)
   }
```

Comment:
Unfortunately no--we need to generate a direct call to memcpy rather than to a value witness; the compiler needs to know the type is BitwiseCopyable in order to know doing so is correct.

Reply:
I see. Nice to know!